doc/kotlin/API.md

```diff
@@ -20,7 +21,7 @@
         - [Conversion](#Conversion)
 
 ## Album and Product Features
-To prepare to load the photos, you'll need the codes below
+To prepare to load the contents, you'll need the codes below
```

Comment:
content is the plural form of content, dont need the s at the end. apples to all the changes